integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/resources/features/bridge.feature

```diff
@@ -4,13 +4,13 @@ Feature: Bridge tests
     Given authenticate against Manager
     When create a new Bridge "mybridge" in cloud provider "aws" and region "us-east-1"
     Then the list of Bridge instances is containing the Bridge "mybridge"
-    And the Bridge "mybridge" is existing with status "provisioning" within 2 minutes
+    And the Bridge "mybridge" is existing with status "ready" within 4 minutes
 
-#  Scenario: Delete a Bridge
-#    Given authenticate against Manager
-#    And create a new Bridge "mybridge" in cloud provider "aws" and region "us-east-1"
-#    And the list of Bridge instances is containing the Bridge "mybridge"
-#    And the Bridge "mybridge" is existing with status "provisioning" within 2 minutes
-#    When delete the Bridge "mybridge"
-#    Then delete the Bridge "mybridge" is failing with HTTP response code 400
+  Scenario: Delete a Bridge
```

Comment:
One idea - considering that the `Create a Bridge` and `Delete a Bridge` scenarios test moreless the same it could be good to merge those tests into one.
That will speed up test execution in future.

Reply:
yep agree with that and will do. Thank you